services/app-api/.gitignore

```diff
@@ -53,4 +53,4 @@ env.yml
 
 # Serverless warmup plugin temp dir
 _warmup
-.warmup
+.warmup
```

Comment:
This one is still showing as missing the newline

Reply:
this should be good now